example.exs

```diff
@@ -136,7 +136,31 @@ sheet6 =
   # nest further
   |> Sheet.group_cols("C", "D")
 
+
+# Autofilter can be applied to allow filtering a range
+sheet7 =
+  %Sheet{
+    name: "Autofilter columns",
+    rows: [
+      [1, 2, 3],
+      [4, 5, 6],
+      [7, 8, 9],
+      [10, 11, 12],
+      [13, 14, 15],
+    ]
+  }
+  # first you select the filtering range
```

Comment:
What happens if a column has an autofilter, but no list_filter or criteria_filter applied?

Reply:
First of all, I would like to clear up a misconception. An autofilter does not apply to a single column, but a range. E.g. assume you have names, postal codes, and adresses in column A, B, and C respectively. Then a filtering based on say a postal code applies to all three columns, if they are all selected as the filtering range. 

Next, to answer your question. In Excel if you create a filtering range without selecting any filter, you will see dropdown carets on each column in the filtering range (for selecting a filter obviously), but with no filter applied yet. No harm done, and it may actually be beneficial, as you might want to generate a worksheet with a filtering range but no filters applied as a convenience for your users.